bot/discord.js

```diff
@@ -11,30 +18,48 @@ useProxy(PROXY_URL)
 useCommandsCreate(BOT_TOKEN, CLIENT_ID, GUILD_ID)
 
 const { Client, Intents } = require('discord.js')
+const { potentialScammerModule } = require(
+  './potentialScammer/potentialScammerModule')
 const client = new Client(
-  { intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] })
+  {
+    intents: [
+      Intents.FLAGS.GUILDS,
+      Intents.FLAGS.GUILD_MESSAGES,
+      Intents.FLAGS.GUILD_MEMBERS],
+  })
 
 client.on('ready', () => {
   console.log(`Logged in as ${client.user.tag}!`)
+  setInterval(function () {
+    potentialScammerModule(client)
+  }, 100)
 })
 
 client.on('interactionCreate', async interaction => {
+  //it is should a command
   if (!interaction.isCommand()) return
-
-  if (interaction.commandName === 'ping') {
-    await interaction.reply('Pong!')
-  }
-
-  if (interaction.commandName === 'hello') {
-    await interaction.reply('Nice!')
-  }
+  //What should be done after the command is executed
+  await commandReaction(interaction)
 })
 
 client.on('messageCreate', async (message) => {
+  //message that should be ignore
   ignoreMessage(message)
-  if (message.mentions.has(client.user.id)) {
-    await message.reply('Hello')
+  //The reaction to  particular messages
+  await messageReaction(message)
+})
+
+client.on('guildCreate', async (server) => {
```

Comment:
How about check if the channel exists and create one if not after every login, in `on('ready', ...)`.